openstack/nova/templates/etc/_nova-scheduler.conf.tpl

```diff
@@ -1,23 +1,23 @@
 [DEFAULT]
 # Scheduling
-scheduler_driver_task_period = {{ .Values.scheduler.driver_task_period | default 60 }}
 scheduler_host_manager = {{ if .Values.global.hypervisors_ironic }}ironic_host_manager{{ else }}host_manager{{ end }}
 scheduler_driver = {{ .Values.scheduler.driver }}
-scheduler_available_filters = {{ .Values.scheduler.available_filters | default "nova.scheduler.filters.all_filters" }}
-scheduler_default_filters = {{ .Values.scheduler.default_filters}}
-
-cpu_weight_multiplier = {{ .Values.scheduler.cpu_weight_multiplier }}
-ram_weight_multiplier = {{ .Values.scheduler.ram_weight_multiplier }}
-disk_weight_multiplier =  {{ .Values.scheduler.disk_weight_multiplier }}
-io_ops_weight_multiplier = {{ .Values.scheduler.io_ops_weight_multiplier }}
-soft_affinity_weight_multiplier = {{ .Values.scheduler.soft_affinity_weight_multiplier }}
-prefer_same_host_resize_weight_multiplier = {{ .Values.scheduler.prefer_same_host_resize_weight_multiplier }}
-scheduler_tracks_instance_changes = {{ .Values.scheduler.scheduler_tracks_instance_changes }}
 
 [scheduler]
 discover_hosts_in_cells_interval = 60
 workers = {{ .Values.scheduler.workers | default 1 }}
+driver_task_period = {{ .Values.scheduler.driver_task_period | default 60 }}
 
 [filter_scheduler]
+available_filters = {{ .Values.scheduler.available_filters | default "nova.scheduler.filters.all_filters" }}
+default_filters = {{ .Values.scheduler.default_filters}}
+tracks_instance_changes = {{ .Values.scheduler.tracks_instance_changes }}
```

Comment:
This was `.Values.scheduler.scheduler_tracks_instance_changes` before, but I see now values file change.

Reply:
Updated the value.